cli/dts/lib.deno.unstable.d.ts

```diff
@@ -130,6 +130,31 @@ declare namespace Deno {
    */
   export function osRelease(): string;
 
+  /** **Unstable** new API. yet to be vetted. 
+   *
+   * Returns the release version of the Operating System.
+   *
+   * ```ts
+   * console.log(Deno.memoryInfo());
+   * ```
+   *
+   * Requires `allow-env` permission.
+   *
+   */
+  export function memoryInfo(): MemoryInfo;
+
+  export interface MemoryInfo {
+    /** Total physical memory. */
+    total: number;
+    free: number;
+    available: number;
+    buffers: number;
+    cached: number;
```

Comment:
Same here. What's `buffers` and what is `cached`, I would think those are the same thing too.